DFC.App.FindACourse/Startup.cs

```diff
@@ -32,7 +43,31 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IFindACourseService, FindACourseService>();
             services.AddScoped<IFindACourseRepository, FindACourseRepository>();
 
+            var courseSearchSettings = Configuration.GetSection(CourseSearchAppSettings).Get<CourseSearchSettings>();
+            services.AddSingleton(courseSearchSettings ?? new CourseSearchSettings());
+
+            var courseSearchClientSettings = new CourseSearchClientSettings
+            {
+                CourseSearchSvcSettings = Configuration.GetSection(CourseSearchClientSvcSettings).Get<CourseSearchSvcSettings>() ?? new CourseSearchSvcSettings(),
+                CourseSearchAuditCosmosDbSettings = Configuration.GetSection(CourseSearchClientAuditSettings).Get<CourseSearchAuditCosmosDbSettings>() ?? new CourseSearchAuditCosmosDbSettings(),
+                PolicyOptions = Configuration.GetSection(CourseSearchClientPolicySettings).Get<PolicyOptions>() ?? new PolicyOptions(),
+            };
+            services.AddSingleton(courseSearchClientSettings);
+            services.AddScoped<ICourseSearchApiService, CourseSearchApiService>();
+            services.AddFindACourseServices(courseSearchClientSettings);
+
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.AddSingleton(serviceProvider =>
+            {
+                return new MapperConfiguration(cfg =>
+                {
+                    cfg.AddProfiles(
+                        new List<Profile>
+                        {
+                            new FindACourseProfile(),
+                        });
+                }).CreateMapper();
+            });
 
             services.AddSwaggerGen(c =>
```

Comment:
Do we need Swagger for this project?

Reply:
This was there by default.  Removed as there is already one for the FAC API.